packages/acot-preset-wcag/README.md

```diff
@@ -55,9 +56,9 @@ You can also enable all the recommended rules for our preset. Add `preset:@acot/
 
 #### 1.1 Text Alternatives
 
-| Success criteria       | acot rules                                              |
-| ---------------------- | ------------------------------------------------------- |
-| 1.1.1 Non-text Content | [@acot/wcag/img-has-name](./docs/rules/img-has-name.md) |
+| Success criteria       | acot rules                                                                                                                           |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| 1.1.1 Non-text Content | [@acot/wcag/img-has-name](./docs/rules/img-has-name.md)<br />[@acot/wcag/invalid-id-reference](./docs/rules/invalid-id-reference.md) |
```

Comment:
In this Pull Request, this rule cover not only the success criteria that are directly covered, but also the success criteria that cannot be achieved when the id reference fails.

However, I think it would be better to state that the rule only covers the success criteria that it directly covers. (e.g., if the `name` of `input` is not given when the label-for is not related, the input-has-name will fail. So, 1.1.1 was covered by input-has-name.

I think this rule directly covered only 1.3.1, 4.1.1.

Reply:
@masuP9 

Thank you for your review.
You are right, there are indeed some success criteria that are not covered :)

> I think this rule directly covered only 1.3.1, 4.1.1.
OK, I'm going to change to 1.3.1 and 4.1.1 :+1: